src/components/assessment/__tests__/index.tsx

```diff
@@ -12,6 +12,7 @@ const defaultProps: IAssessmentProps = {
   assessmentOverviews: undefined,
   handleAssessmentOverviewFetch: () => {},
   handleSubmitAssessment: (id: number) => {},
+  isStudent: false,
```

Comment:
Should write a test for both `isStudent` values, and make sure that the button does not render for students.

Reply:
Yep, sounds good

Reply:
Default is `false`, so I added a test for when the user is a student